caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -87,7 +87,7 @@ func parseTLS(h Helper) ([]ConfigValue, error) {
 	var folderLoader caddytls.FolderLoader
 	var certSelector caddytls.CustomCertSelectionPolicy
 	var acmeIssuer *caddytls.ACMEIssuer
-	var autoPolicy *caddytls.AutomationPolicy
+	var keyType *string
```

Comment:
Sorry to be pendantic, but why is this a pointer? :sweat_smile:  Probably safer to use a `string` instead. Just check for `keyType != ""`, impossible to panic accidentally.

```suggestion
	var keyType string
```

Reply:
Not a problem. I'm new to golang :)